src/features/MovieList/MovieTile/styled.js

```diff
@@ -66,27 +68,31 @@ export const Year = styled.p`
     }
 `;
 
-export const TagsContainer = styled.div`
+export const TagsContainer = styled.ul`
     margin-top: 8px;
     display: flex;
     flex-wrap: wrap;
     gap: 8px;
```

Comment:
If someone want to use our browser on 176px this solve problem going over the edge of container.
i know There are no such screens 176px xd, but if there is a 100% responsive option, why not

Reply:
You are completely right, if we can make it better and working on more screens, possibly also not existing screens we should pursue it. Some people divide their screens into more applications so it is theoretically possible that someone would want to browse our page in browser that is shrinked so that the viewport is 176px wide.

Reply:
yes flex-wrap making this super responsive